src/main/java/io/jenkins/plugins/opentelemetry/OpenTelemetrySdkProvider.java

```diff
@@ -119,9 +117,7 @@ public void initializeForGrpc(@Nonnull String endpoint, @Nullable String trusted
 
         preDestroy();
 
-        // TODO variabilize
-        int timeoutMillis = 30_000;
-        int exportIntervalMillis = 60_000;
+        int timeoutMillis = JenkinsOpenTelemetryPluginConfiguration.get().getCollectorTimeout();
```

Comment:
Can we keep the isolation and not have dependencies on `JenkinsOpenTelemetryPluginConfiguration` in `OpenTelemetrySdkProvider`? 

I guess it means passing the metrics exportInterval as a parameter of `io.jenkins.plugins.opentelemetry.OpenTelemetrySdkProvider#initializeForGrpc()`

Reply:
See https://github.com/jenkinsci/opentelemetry-plugin/pull/39/commits/353721f6449bcedad4e462f0245cca20d67a70b9